app/templates/admin/createEvent.html

```diff
@@ -161,18 +161,34 @@ <h1 id="title">{{page_title}}</h1>
               </div>
           </div>
             {% if page_title != 'Create All Volunteer Training' %}
-              <div class="form-check form-switch pt-4">
-                <label class="custom-control-label" for="checkIsTraining"><strong>This event is a training.</strong></label>
-                <input class="form-check-input" type="checkbox" id="checkIsTraining" name="isTraining" {{"checked" if eventData.isTraining}}/>
-              </div>
-              <div class="form-check form-switch">
-                <label class="custom-control-label" for='rsvp'><strong>This event requires RSVP.</strong></label>
-                <input class="form-check-input" type="checkbox" id="rsvp" name="isRsvpRequired" {{"checked" if eventData.isRsvpRequired}}/>
-              </div>
-              <div class="form-check form-switch mb-4">
-                <label class="custom-control-label" for='earnServiceHours'><strong>This event earns service hours.</strong></label>
-                <input class="form-check-input" type="checkbox" id="earnServiceHours" name="isService" {{"checked" if eventData.isService}}/>
+            <div class="pt-4 mb-4">
+              <div class="row col-md-12">
+                <div class="form-group col-md-6">
+                  <div class="form-check form-switch mb-2">
+                    <label class="custom-control-label" for="food"><strong>Food is provided.</strong></label>
```

Comment:
I think it would be nice if we could try and have this label match the others. Maybe something like "This event will have food provided." or "This event will provide food."